.babelrc

```diff
@@ -1,8 +1,6 @@
 {
-	"sourceMap": "inline",
-	"presets": ["es2015-loose", "stage-0", "react"],
+	"presets": ["@babel/env", "@babel/react"],
```

Comment:
Babel officially deprecated stage-0. Though @babel/preset-env is less specific here than the old preset was, I believe a) the fact that this library was written in a very future-thinking method, and b) browsers just support a lot more now, I think this functionally works out to the same output. This is just the now-supported way to do it.